grails-app/views/attribute/edit.gsp

```diff
@@ -94,6 +94,14 @@
                                     </a>
 								</td>
 							</tr>
+                            <tr class="prop">
+                                <td valign="top" class="name">
+                                    <label for="entityTypeCodes"><g:message code="entityTypeCodes.label" default="Entity Type Codes" /></label>
+                                </td>
+                                <td valign="top" class="value ${hasErrors(bean: attributeInstance, field: 'entityTypeCodes', 'errors')}">
+                                    <g:selectEntityTypeCode name="entityTypeCodes" value="${attributeInstance?.entityTypeCodes}" class="chzn-select-deselect"/>
```

Comment:
On edit, there is a multi-select, but when creating it is as a single-select. Should this be unified?

Reply:
I initially had it as a multipe but realized there were edge cases I didn't want to handle.